jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -70,17 +70,35 @@
 	private static final class MethodRefsHolder {
 		static Method startLocalManagementAgentMethod = null;
 		static Method startRemoteManagementAgentMethod = null;
+		static final Throwable managementAgentMethodThrowable;
 		static {
-			AccessController.doPrivileged((PrivilegedAction<Object>) () -> {
-				Class<?> agentClass;
-				Class<?> startRemoteArgumentType;
+			managementAgentMethodThrowable = (Throwable) AccessController.doPrivileged((PrivilegedAction<Object>) () -> {
```

Comment:
The action should be of type `PrivilegedAction<Throwable>` so the cast need not be explicit.